dart/lib/src/platform_checker.dart

```diff
@@ -17,4 +19,18 @@ class PlatformChecker {
   bool isProfileMode() {
     return const bool.fromEnvironment('dart.vm.profile', defaultValue: false);
   }
+
+  /// This can be set as [SentryOptions.environment]
+  String get environment {
```

Comment:
should this be a method instead? quite a few conditions to be just a property, maybe we move hardcoded values to `EnvironmentVariables` instead